agent/src/softwarecontaineragentadaptor.cpp

```diff
@@ -0,0 +1,58 @@
+#include "softwarecontaineragentadaptor.h"
+
```

Comment:
No, but in other places. Function lines are a special case. according to our coding guideline.